### PR TITLE
Add example with-purgecss

### PR DESCRIPTION
**Pull request by Ffloriel**
# Add example with purgecss and tailwindcss

[Purgecss](https://github.com/FullHuman/purgecss) is a tool to remove unused CSS. It can be used as part of your development workflow. Purgecss comes with a JavaScript API, a CLI, and plugins for popular build tools.

This tool is often used with tailwindcss. The PR adds an example of a nuxtjs configuration with purgecss and tailwindcss

## Related issues

https://github.com/FullHuman/purgecss/issues/54
https://github.com/nuxt/nuxt.js/issues/892
https://github.com/nuxt/nuxt.js/issues/2810

### CODE REVIEW

File: examples/with-purgecss/tailwind.js [hunk 1]
```diff
@@ -0,0 +1,744 @@
+/*
+
+Tailwind - The Utility-First CSS Framework
+
+A project by Adam Wathan (@adamwathan), Jonathan Reinink (@reinink),
+David Hemphill (@davidhemphill) and Steve Schoger (@steveschoger).
+
+Welcome to the Tailwind config file. This is where you can customize
+Tailwind specifically for your project. Don't be intimidated by the
+length of this file. It's really just a big JavaScript object and
```
Comment (qm3ster) [inline examples/with-purgecss/tailwind.js]: GitHub diff sure got intimidated by the length of this file :rofl:

**Comment by qm3ster** [comment]
And now, the big question: should examples in the main repo have tests?
:thinking:

File: examples/with-purgecss/tailwind.js (hunk 1 above)
Comment (husayt) [reply examples/with-purgecss/tailwind.js]: @qm3ster indeed, it is better to include it through package.json and remove tailwind.js from project's repo files.

Comment (Ffloriel) [reply examples/with-purgecss/tailwind.js]: @husayt I based this example on the tailwindcss one. The tailwind.js is the config file. https://tailwindcss.com/docs/installation#2-create-a-tailwind-config-file

Comment (husayt) [reply examples/with-purgecss/tailwind.js]: I see. Then I don't see a problem. Thanks for useful example.

File: examples/with-purgecss/nuxt.config.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+              path.join(__dirname, './pages/**/*.vue'),
+              path.join(__dirname, './layouts/**/*.vue')
+            ]),
+            extractors: [
+              {
+                extractor: TailwindExtractor,
+                extensions: ["vue"]
+              }
+            ],
+            whitelist: ['html', 'body']
```
Comment (manniL) [inline examples/with-purgecss/nuxt.config.js]: Please also whitelist `nuxt-progress` which is used for the progress bar on top. :relaxed:

**Comment by husayt** [comment]
@Ffloriel  any chance you fix build issues? You can see them in details, that is only thing which is preventing merge, as I see.

**Comment by husayt** [comment]
I have just tried to get Purgecss and Nuxt working with vuetify, it worked but some css was lost. 
I followed official example. 
https://github.com/FullHuman/purgecss/tree/master/examples/with-nuxt

Still trying to fix, once found will share

**Comment by Atinux** [comment]
This is awesome! I did not know if purgecss will be able to work with Nuxt.

Will look more into it, this would be great if it could work with others UI frameworks yes. Then find a way to include it in Nuxt, I would love this feature, everyone!

I will look at this example and merge it when verified. Thank you for this PR.

**Pull request merged**